README.md

```diff
@@ -1 +1,3 @@
-# trialreviewtribue
```

Comment:
- [ ] I think here you can say it in a polite tone and with saying the reason why you are asking for this change because maybe it is obvious to you but not to others💪🏻💪🏻

so instead of saying:

> Add description below

you can say:

> I think you should add a description here because this displays a small summary of the project and it is important in the future for the recruiters to see about your projects easily and to outstand from the other applicants.🙏🏻✅